docs/source/configuration.rst

```diff
@@ -43,6 +43,18 @@ expected to use upper-case letters and must be prefixed with
    - ``aws``
    - ``azure``
 
+
+   The operator will attempt to set the relevant ``node.attr.zone`` parameter on CrateDB based on the cloud provider,
+   however in some cases this might need to be overridden. This can be done with:
+
+   .. code-block:: yaml
+
+      kind: CrateDB
+      spec:
+        cluster:
+          settings:
+            node.attr.zone: "$(curl -s 'https://example.com')"
+
```

Comment:
This sounds good already! Some suggestions/rewordings, though.

restructuredText files typically have a line length of 79 characters. Would be nice to align with that here as well, to keep the docs consistent.

```suggestion
   Under the hood, the operator will pass a ``zone`` attribute to all CrateDB
   nodes. This attribute can also be defined explicitly or override the one set
   by the operator. To do this on a cluster level, set
   ``.spec.cluster.settings``:

   .. code-block:: yaml

      kind: CrateDB
      spec:
        cluster:
          settings:
            node.attr.zone: "some-value"

   To set or override the attribute on a node type level, set it in
   ``.spec.nodes.master.settings`` or ``.spec.nodes.data.*.settings``.
```